auth-sidecar/Dockerfile

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright 2018 Astronomer Inc.
+#
+# Licensed under the Apache License, Version 3.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# https://hub.docker.com/r/nginxinc/nginx-unprivileged/tags
+FROM nginxinc/nginx-unprivileged:1.21.4-alpine
+LABEL maintainer="Astronomer <humans@astronomer.io>"
+
+ARG BUILD_NUMBER=-1
+LABEL io.astronomer.docker.build.number=$BUILD_NUMBER
+LABEL io.astronomer.docker.module="default-backend"
+LABEL io.astronomer.docker.component="default-backend"
```

Comment:
@danielhoherd - we need change the naming

Reply:
just blindly changed from default-backend to auth-sidecar, let me know if that wasn't the right thing to do, haven't looked into what these are used for